packages/fxa-content-server/app/scripts/templates/settings/client_disconnect.mustache

```diff
@@ -1,14 +1,16 @@
 <div id="client-disconnect">
   <section class="modal-panel">
     {{^hasDisconnected}}
-      <h2>{{#t}}Disconnect from Sync?{{/t}}</h2>
+      <h2>{{#t}}Disconnect from Sync{{/t}}</h2>
       <div class="error"></div>
 
       <p class="intro">
       {{#t}}Your browsing data will remain on %(deviceName)s, but it will no longer sync with your account.{{/t}}
       </p>
 
       <form novalidate>
+        <div role="radiogroup" aria-labelledby="radio-header">
+          <h3 id="radio-header" class="radio-header">{{#t}}You are disconnecting %(deviceName)s from Sync because this device is:{{/t}}</h3>
```

Comment:
The wording "this device is" makes the radio button labels seem off, .e.g.:

"this device is:" "Suspicious device"

The "device" on the end of "suspicious" seems somewhat superfluous.

Reply:
@janiceahn @janiceahn Agreed, I didn't notice this. Would you like to tweak this copy?